src/assets/translations/messages-en.json

```diff
@@ -12,6 +12,7 @@
     "noResultsFound": "No results found.",
     "insufficientFunds": "Insufficient Funds",
     "invalidAddress": "Invalid Address",
+    "unresolvableEnsDomain": "Cannot resolve ENS domain",
```

Comment:
The reason why the terminology is different than the `ENS Name` in different places is because it's matching the string. Why is the string `ENS domain` vs. `ENS name`? Well, from a user standpoint, `ENS domain` is clearer IMO, `name` is the more technical terminology.